reference/batch-workflow-syntax.md

```diff
@@ -189,6 +204,172 @@ cache:
 
 **Expression contexts:** [`flow` context](batch-contexts.md#flow-context), [`params` context](batch-contexts.md#params-context), [`env` context](batch-contexts.md#env-context), [`tags` context](batch-contexts.md#tags-context), [`volumes` context](batch-contexts.md#volumes-context), [`images` context](batch-contexts.md#images-context).
 
+## `images`
+
+A mapping of image definitions used by the _batch_ workflow.
+
+If the specified image reference is available at the Neu.ro registry and the [`.force_rebuild`](batch-workflow-syntax.md#images-less-than-image-id-greater-than-force-rebuild) flag is not set, then Neu.ro Flow starts building the image from scratch. 
+If this flag is set, then the existing image from the registry will be used.
+
+{% hint style="info" %}
+The `images` section is not required. A task can specify the image name in a plain string without referring to the `${{ images.my_image.ref }}` context.
+
+However, this section exists for convenience: there is no need to repeat yourself if you can just point the image reference everywhere in the YAML.
```

Comment:
This block adds new functionality (in batch mode), so I'm not sure about this.

Reply:
Fixed in the recent commit